conduit/data/datasets/audio/ecoacoustics.py

```diff
@@ -266,8 +266,10 @@ def load_sample(self, index: int) -> Tensor:
             self.num_frames_in_segment / self.sample_rate * metadata.sample_rate
         )
 
-        # compute offset
-        high = max(1, metadata.num_frames - self.num_frames_in_segment)
+        # when the number of frames in the data is greater than the
+        # specified number of frames, apply i.i.d. sampling, slice off
+        # a randomised portion of the sample using an offset
```

Comment:
Comment doesn't add more information than the code. I'd get rid of it. If you're desperate to keep it, maybe: 
```suggestion
        # get random sub-sample
```